### ISSUE TITLE
new LICENSE file incorrect, and would not be detected by github

### ISSUE THREAD

**Issue by mwichmann** [github/docs#275]
### What article on docs.github.com is affected?

None

### What part(s) of the article would you like to see updated?

None

### Additional information

Not a problem of displayed content, but of a file in the repo:

The LICENSE file (recently promoted via #204) is an overly-complete paste from Creative Commons.  It includes the preamble (with a bit missing) and afterword, which are intended as instructions to users, and are not part of the license text itself. Lines 1-56 and lines 367-end can (should?) be dropped - search for the ===== delimiters.

In addition, if you select this license, github does not recognize the license when it presents summary information the way it usually does for the project as a whole, and if you click on a LICENSE file (independently tested on a fresh repo since this one is "incorrect" as noded).  That's not an error of the docs repo itself, of course, but at least can make the team aware of it.

**Comment by zeke** [comment]
Thanks for the report, @mwichmann 👍🏼 

cc @jeffmcaffer who expressed an interest in making sure our licensees correct, discoverable, etc.

**Comment by jeffmcaffer** [comment]
Thanks @mwichmann. Indeed the LICENSE content is overly complete (love how you put that). Having said that, licensee, the underlying tool that identifies licenses is working as expected and does detect both the CC and MIT licenses. (see output below). The challenge is that the GitHub UI does not deal with multiple licenses and instead opts to say View license and point users at the LICENSE file. 

So for this issue we should likely invoke the lawyers to ensure we have the right file content and pursue the multi-license handling in the UI separately.

```
~/git/test$ licensee detect --remote https://github.com/github/docs
License:        NOASSERTION
Matched files:  LICENSE, LICENSE-CODE, README.md, package.json
LICENSE:
  Content hash:  25b636566bb4cee96f8e917e6382b7c4a6858a90
  Confidence:    100.00%
  Matcher:       Licensee::Matchers::Exact
  License:       CC-BY-4.0
LICENSE-CODE:  
  Content hash:  d64f3bb4282a97b37454b5bb96a8a264a3363dc3
  Attribution:   Copyright (c) 2020 GitHub
  Confidence:    100.00%
  Matcher:       Licensee::Matchers::Exact
  License:       MIT
README.md:
  Content hash:  7c25f06c9b30aca9c1da23ad86968bb38db2d53f
  Confidence:    90.00%
  Matcher:       Licensee::Matchers::Reference
  License:       MIT
  Closest non-matching licenses:
    WTFPL similarity:  16.22%
    0BSD similarity:   15.50%
    ISC similarity:    13.89%
package.json:
  Confidence:  90.00%
  Matcher:     Licensee::Matchers::NpmBower
  License:     NOASSERTION
```

**Comment by lee-dohm** [comment]
Thanks for bringing this to our attention. We've indeed consulted with our legal department and we are following the best practices for documenting our licensing information within the repository. Additionally, as @jeffmcaffer points out above, our licensee tool detects both licenses properly when run on the repository directly. Since we've confirmed that we're following the best practices we're going to go ahead and close this.

Thanks very much for your interest and passion in the GitHub docs! ✨
